admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
+![](img/notifications-updates.png)
+- Updates for the cluster or for specific operators could also be listed here.
+- Clicking on one of these cards would take the user to a page with information about the updates.
+- Action buttons to install the update immediately could also be provided.
+
+## Empty State
+![](img/notifications-empty.png)
+- When there are no critical alerts firing, the `Critical alerts` group should still be present but should contain an empty state pattern.
+- The `Available updates` category and any additional categories should be hidden when they are empty.
+
+## First Time Setup
+![](img/notifications-receivers.png)
+- Potentially, alerts with a specific annotation could have the annotation rendered as an action or link button. Otherwise, clicking on the alert itself would take the user to the details page for the alert.
+- A special severity level for “getting started” or “first time setup” alerts could cause them to be rendered in a special way.
```

Comment:
For now we can just hardcode by alert name.

Reply:
Is this section also intended for the cluster admin/specific roles?